nntrainer/layers/fc_layer.cpp

```diff
@@ -66,17 +66,22 @@ void FullyConnectedLayer::finalize(InitLayerContext &context) {
   context.setEffDimFlagInputDimension(0, 0b1001);
   context.setDynDimFlagInputDimension(0, 0b1000);
 
+  bool is_nchw = (getTensorType() == Tformat::NCHW) ? true : false;
```

Comment:
```( a == b ) ? true : false```
==
``` a == b ```

Reply:
Removed the unwanted ternary operators here and in [tensor.cpp](https://github.com/nnstreamer/nntrainer/pull/2246/files#diff-8a77082124d610f5fb8ab7e7cc4efab8ca088a9f98ecf8c5288cfbecd55d91f8). Thank you.